packages/beacon-node/src/db/repositories/dataColumnSidecars.ts

```diff
@@ -0,0 +1,49 @@
+import {ByteVectorType, ContainerType, ValueOf} from "@chainsafe/ssz";
+import {ChainForkConfig} from "@lodestar/config";
+import {Db, Repository} from "@lodestar/db";
+import {NUMBER_OF_COLUMNS} from "@lodestar/params";
+import {ssz} from "@lodestar/types";
+
+import {Bucket, getBucketNameByValue} from "../buckets.js";
+
+export const dataColumnSidecarsWrapperSsz = new ContainerType(
+  {
+    blockRoot: ssz.Root,
+    slot: ssz.Slot,
+    dataColumnsLen: ssz.Uint8,
+    dataColumnsSize: ssz.UintNum64,
+    // // each byte[i] tells what index (1 based) the column i is stored, 0 means not custodied
+    // max value to represent will be 128 which can be represented in a byte
+    dataColumnsIndex: new ByteVectorType(NUMBER_OF_COLUMNS),
+    dataColumnSidecars: ssz.fulu.DataColumnSidecars,
+  },
+  {typeName: "DataColumnSidecarsWrapper", jsonCase: "eth2"}
+);
+
+export type DataColumnSidecarsWrapper = ValueOf<typeof dataColumnSidecarsWrapperSsz>;
+export const BLOCK_ROOT_IN_WRAPPER_INDEX = 0;
+export const BLOCK_SLOT_IN_WRAPPER_INDEX = 32;
+export const NUM_COLUMNS_IN_WRAPPER_INDEX = 40;
+export const COLUMN_SIZE_IN_WRAPPER_INDEX = 41;
+export const CUSTODY_COLUMNS_IN_IN_WRAPPER_INDEX = 49;
+export const DATA_COLUMN_SIDECARS_IN_WRAPPER_INDEX = CUSTODY_COLUMNS_IN_IN_WRAPPER_INDEX + NUMBER_OF_COLUMNS + 4;
```

Comment:
Suggest to name this magic number `4` as `LIST_SIZE_PREIX_BYTES` so it's much easy to read.